File: ympa_fsharp_giraffe_server/src/model/WarningPromoOfferUpdateDTO.fs
namespace ympa_fsharp_giraffe_server.Model

open System
open System.Collections.Generic
open ympa_fsharp_giraffe_server.Model.PromoOfferUpdateWarningDTO

module WarningPromoOfferUpdateDTO =

  //#region WarningPromoOfferUpdateDTO


  type WarningPromoOfferUpdateDTO = {
    OfferId : string;
    Warnings : PromoOfferUpdateWarningDTO[];
  }
  //#endregion

File: src/main/kotlin/ympa_kotlin_wiremock_server/models/WarningPromoOfferUpdateDTO.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class WarningPromoOfferUpdateDTO(
    @field:JsonProperty("offerId")
    val offerId: kotlin.String,

    @field:JsonProperty("warnings")
    val warnings: kotlin.collections.List<PromoOfferUpdateWarningDTO>,

)
